src/recoil/app-theme/app-theme.state.ts

```diff
@@ -19,15 +20,16 @@ const isWithinRange = (date: DateTime<true> | DateTime<false>) => {
   return now >= date.minus({ days: 13 }) && now <= date.plus({ days: 1 })
 }
 
-const halloween = DateTime.fromObject({ month: 10, day: 31 }).endOf(`day`)
+//const halloween = DateTime.fromObject({ month: 10, day: 31 }).endOf(`day`)
```

Comment:
dont comment this out either!

Reply:
Handled in [8ef685e](https://github.com/ajktown/ConsistencyGPT/pull/74/commits/8ef685e54a10fbb3a6fff4f9964b248e91c083eb)